app/src/views/search-engine/components/search-bar/index.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { AppBar, Toolbar, Avatar, Input } from '@material-ui/core';
+import styles from './styles';
+
+const SearchBar = ({ setSearch }) => {
+  const classes = styles();
+
+  return (
+    <AppBar position="sticky" color="inherit" className={classes.appbar}>
+      <Toolbar>
+        <div className={classes.search}>
+          <Input
+            type="search"
+            placeholder="Buscar..."
+            classes={{
+              root: classes.inputRoot,
+              input: classes.inputInput,
+            }}
+            onChange={({ target: { value } }) => setSearch(value)}
```

Comment:
Could we also please add an actual button to trigger the search. It is not inherent for everyone to click the enter key. It helps greatly in terms of accessibility for the elderly and visually challenged people the addition of buttons. I know we are not that focused on this demographic but a simple button is a good investment in terms of time v.s. accessibility created.

Reply:
Sure, although, the second you start typing, the search bar is ready to perform a query, so, the second you finish typing, the request is automatically triggered, there's no need for any manual triggering, rendering the suggested button useless.

Reply:
This is true for many apps, that the searches are done automatically, or at least suggested automatically, but all of this apps still have some kind of button to press to force the search. Maybe put it inside the text box but translucent?

Reply:
Yeah, I'll be adding a search icon to force any request.